tcms/templates/run/get_case_runs.html

```diff
@@ -156,20 +156,15 @@
 				{{/each}}
 			<div class="bug_type" id="id_bug_type">Issue Tracker</div>
 			<select name="bug_system_id" id="bug_system_id">
-				<option value="1" selected="selected" data-tab="bug-system-bugzilla">Bugzilla</option>
-				<option value="2" data-tab="bug-system-jira">JIRA</option>
+				{{#each bugTrackers}}
+					<option value="{{ @key }}">{{ this }}</option>
+				{{/each}}
 			</select>
 			</br>
-			<div id="bug-system-bugzilla" style="line-height:30px;">
+			<div id="bug-system" style="line-height:30px;">
 				<div class="bug_id">Bug ID</div>
 					<input type="text" name="bug_id">
 					</br>
-					<input type="checkbox" name="bz_external_track" value="bz_external_track">&nbsp;
-					<span class="use_bz_external_track">Check to add Test Cases to BZ</span>
```

Comment:
There is a bigger problem with this checkbox than disabling via setting or removing it from the code. Currently it works only for Red Hat Bugzilla, possibly another Bugzilla instance. 
Because this PR removes everything that is hard-coded we don't really know how the remote system works anymore. 

I imagine it will require a few more settings - the remote URL to receive the request and maybe a callback which knows how to send said request.   We need to provide a very flexible way for users to be able to integrate Nitrate with whatever bug tracking system they want to use. To make it easier we can provide some default integrations for Bugzilla, JIRA, etc.

I will open a new issue to track this but will not add the changes to this PR because it is already very big.